.github/workflows/create-cli-release.yml

```diff
@@ -91,7 +91,7 @@ jobs:
         with:
           max_attempts: 5
           retry_wait_seconds: 120
-          command: sf-release cli:versions:inspect -c ${{ needs.get-channel.outputs.s3-channel }} -l archive --cli sf
+          command: sf-release cli:versions:inspect -c ${{ needs.get-channel.outputs.s3-channel }} -l archive
```

Comment:
This _shouldn't_ need the `--ignore-missing` logic because it only checks the channel it just created